.build/build-resolver.xml

```diff
@@ -165,7 +165,7 @@
         <property name="resolver-ant-tasks.initialized" value="true"/>
     </target>
 
-    <target name="resolver-retrieve-build" depends="resolver-init,write-poms">
+    <target name="resolver-retrieve-build" depends="resolver-init,write-poms,_build_subprojects">
```

Comment:
i'm hesitant about this. i'm curious if it is better this is added one-level up to those that call `resolver-retrieve-build`.

there's the nit wrt encapsulation. but i'm more concerned about other scripts that call this target solely to cache the network downloading (e.g. our docker images). if they start doing actual building that's a bit confusing…

Reply:
this is what I see

```
$ grep -r resolver-retrieve-build build.xml .build/*
build.xml:    <target name="build" depends="resolver-retrieve-build,build-project,checkstyle" description="Compile Cassandra classes"/>
build.xml:    <target name="_build_unsafe" depends="resolver-retrieve-build,build-project" description="Compile Cassandra classes without checks"/>
build.xml:  <target name="init-checkstyle" depends="resolver-retrieve-build,build-project" unless="no-checkstyle">
build.xml:  <target name="checkstyle-test" depends="init-checkstyle,resolver-retrieve-build,build-project" description="Run custom checkstyle code analysis on tests" if="java.version.8" unless="no-checkstyle">
.build/build-resolver.xml:    <target name="resolver-retrieve-build" depends="resolver-init,write-poms,_build_subprojects">
.build/build-resolver.xml:    <target name="resolver-dist-lib" depends="resolver-retrieve-build">
```

ill test this out

Reply:
pushed the change will see how CI works; if everything is good will resolve this (and we can cherry pick which tasks do not depend)